diff --git a/node_modules/pdf-parse/lib/pdf.js b/node_modules/pdf-parse/lib/pdf.js
index 45a6a68..f0a1c1d 100644
--- a/node_modules/pdf-parse/lib/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js
@@ -1,4 +1,4 @@
-const { getDocument, PDFDataRangeTransport } = require('../pdf.js/build/pdf.js');
+const { getDocument, PDFDataRangeTransport } = require('pdfjs-dist/legacy/build/pdf.js');
 
 function PDF(dataBuffer, options) {
 
diff --git a/node_modules/pdf-parse/lib/pdf-parse.js b/node_modules/pdf-parse/lib/pdf-parse.js
index 73562a1..3524b91 100644
--- a/node_modules/pdf-parse/lib/pdf-parse.js
+++ b/node_modules/pdf-parse/lib/pdf-parse.js
@@ -1,4 +1,4 @@
-const { getDocument, PDFDataRangeTransport } = require('../pdf.js/build/pdf.js');
+const { getDocument, PDFDataRangeTransport } = require('pdfjs-dist/legacy/build/pdf.js');
 
 const DEFAULT_OPTIONS = {
     // internal page parser callback
diff --git a/node_modules/pdf-parse/package.json b/node_modules/pdf-parse/package.json
index 10a1122..46c9869 100644
--- a/node_modules/pdf-parse/package.json
+++ b/node_modules/pdf-parse/package.json
@@ -2,7 +2,7 @@
   "name": "pdf-parse",
   "version": "1.1.1",
   "description": "Pdf parser that works in nodejs and browser",
-  "main": "index.js",
+  "main": "lib/pdf-parse.js",
   "keywords": [
     "pdf",
     "parser",
@@ -21,7 +21,8 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "node-ensure": "0.0.0"
+    "node-ensure": "0.0.0",
+    "pdfjs-dist": "2.6.347"
   },
   "devDependencies": {
     "eslint": "^6.8.0",
